welcome.html

```diff
@@ -1,14 +1,23 @@
 {% extends 'base.html' %}
 {% block body %}
-<h1>Welcome!</h1>
-<p>This is a site where students can request their favorite courses.</p>
-<p>Status: Not registered</p>
-<p>Think you should have access? Look at the top right.
-Are you logged in with the correct email address?
-Try clicking on the Selectives Web heading at the top left.</p>
-<p>If you are a student then you'll be able to make your selections
-once an administrator sets up access for you.</p>
-<p>Registration for 2015-2016 Semester One has not started. Please check back on <b>December 26, 2015</b>.</p>
+<h1>Welcome to Discovery Charter School Selectives!</h1>
+
+<p>If you are a student, you may view the course catalog after logging in.</p>
+<ol>
+  <li>Use your mydiscoveryk8.org account.</li>
```

Comment:
Put a login button somewhere here? You can probably just copy it from base.html.